src/jest.js

```diff
@@ -0,0 +1,30 @@
+// @ts-check
+
+import jestPlugin from 'eslint-plugin-jest';
+import {ERROR, testFiles} from './config.js';
+
+/**
+ * @type {Array<import('eslint').Linter.Config>}
+ */
+export default [
+  {
```

Comment:
is `it.only` error already activated in the `estlint-plugin-jest`

Reply:
That's the `no-focused-tests` rule below.